src/components/FeatureViews/Scheduler/EditSessionView.js

```diff
@@ -1,359 +1,660 @@
+    })
+  );
+
+  const instructorList = instructorsData.instructors.map((instructor) => ({
+    value: instructor.user.id,
+    label: `${fullName(instructor.user)} - ${instructor.user.email}`,
+  }));
+
+  const courseDurationOptions = [1, 1.5, 2, 0.5];
+
+  const course = data.session.course;
+
+  const session = data.session;
+
+  return (
+    <Grid container className="main-session-view">
+      <BackgroundPaper
+        elevation={2}
+        className="paper session"
+        mt="2em"
+        style={{ width: "100%" }}
+      >
+        <Grid className="session-button" item>
+          <BackButton />
+        </Grid>
+        <Grid className="session-view" container direction="row" spacing={2}>
+          <Grid item sm={12}>
+            <TextField
+              fullWidth
+              onChange={handleTextChange("title")}
+              value={sessionFields.title}
+            />
+          </Grid>
+          <Grid
+            align="left"
+            className="session-view-details"
+            container
+            spacing={2}
+            xs={6}
+          >
+            {location.state.allOrCurrent === "all" && (
+              <Grid item xs={6}>
+                <Typography variant="h5"> Subject </Typography>
+                <SearchSelect
+                  className="search-options"
+                  isClearable
+                  onChange={handleCategoryChange}
+                  options={categoriesList}
+                  placeholder="Choose a Category"
+                  value={sessionFields.category}
+                />
+              </Grid>
+            )}
+
+            <Grid item xs={6}>
+              <Typography variant="h5"> Room</Typography>
+              <TextField
+                onChange={handleTextChange("room")}
+                value={sessionFields.room}
+              />
+            </Grid>
+
+            <Grid item xs={12}>
+              <Typography variant="h5"> Instructor </Typography>
+              <SearchSelect
+                onChange={handleInstructorChange}
+                options={instructorList}
+                placeholder="Choose an Instructor"
+                value={sessionFields.instructor}
+              />
+              {location.state.allOrCurrent === "current" && (
+                <FormControl
+                  style={{ marginTop: "20px", marginBottom: "10px" }}
+                >
+                  <InputLabel>Is instructor confirmed?</InputLabel>
+                  <Select
+                    onChange={onConfirmationChange}
+                    value={sessionFields.is_confirmed}
+                  >
+                    <MenuItem value>Yes, Instructor Confirmed.</MenuItem>
+                    <MenuItem value={false}>
+                      No, Instructor is NOT Confirmed.
+                    </MenuItem>
+                  </Select>
+                </FormControl>
+              )}
+            </Grid>
+            {location.state.allOrCurrent === "current" && (
```

Comment:
use the `<EditCurrentSessionField/>` once you make it

Reply:
Component made and implemented!